Analysis/HiggsTauTau/interface/HTTConfig.h

```diff
@@ -131,6 +131,7 @@ struct mc_def {
 		phys14_72X,			// 72X MC
     spring15_74X,      //74X MC 
     fall15_76X,      //76X MC 
+    summer16_80X,
```

Comment:
I thought the reHLT sampes were summer mc samples but I must have been mistaken. The only place this option did anything was n the HTTTriggerFilter2.cc module I added - this was to do studies of different HLT paths. I've removed  summer16_80X for all the code and also similified HTTTriggerFilter2.cc quite a bit as it started basically as a copy of HTTTriggerFilter but most of the pre-existing parts aren't needed. I could also rename HTTTriggerFilter2 since it's name is a little confusing (it doesn't actually do any filtering just adds HLT paths to the event)?